app/packages/operators/src/operators.ts

```diff
@@ -889,6 +904,8 @@ export async function resolveExecutionOptions(
       view: currentContext.view,
       view_name: currentContext.viewName,
       group_slice: currentContext.groupSlice,
+      spaces: currentContext.spaces,
+      workspace_name: currentContext.workspaceName,
```

Comment:
I do see that we have both `view` and `view_name` here, so it is consistent to have both `spaces` and `workspace_name` defined here, and also to use both of them in the implementation of `ExecutionContext.spaces` 👍

Reply:
Yup! This is needed to load a workspace instead of building spaces from raw JSON